### DISCUSSION TITLE
Decompiled Retail Console Games · RetroReversing

### DISCUSSION THREAD

**Discussion by giscus** [RetroReversing/retroReversing#119] [Post Comments]
# Decompiled Retail Console Games · RetroReversing

Learn about RetroReversing, the home of retro game reverse engineering enthusiasts. Get resources, tutorials, and tools for analyzing vintage technology and join the community today!

https://www.retroreversing.com/source-code/decompiled-retail-console-games

**Comment by Fixatron** [comment]
I'd like to submit a decompiled game for Famicom, Transformers Convoy no Nazo. Who do I contact to make a submission?

**Reply by RetroGameDeveloper** [reply]
Hey! You can click the edit button at the top of the page and it will allow creating a Pull Request, we can then accept and merge the change and it will show up. Let me know if you need any help
